kuma/wiki/views/edit.py

```diff
@@ -210,36 +209,54 @@ def edit(request, document_slug, document_locale, revision_id=None):
                 if not rev_form.is_valid():
                     # Was there a mid-air collision?
                     if 'current_rev' in rev_form._errors:
+                        # Make the error message safe so the '<' and '>' don't
+                        # get turned into '&lt;' and '&gt;', respectively
+                        rev_form.errors['current_rev'][0] = mark_safe(rev_form.errors['current_rev'][0])
```

Comment:
It seems like it should be fine to mark them both safe, based on the error messages I've seen. However, it's not necessary for HTML ones, so I'll make it only mark AJAX ones as safe.